pkg/volume/vsphere_volume/vsphere_volume_util.go

```diff
@@ -40,15 +40,22 @@ const (
 	checkSleepDuration = time.Second
 	diskByIDPath       = "/dev/disk/by-id/"
 	diskSCSIPrefix     = "wwn-0x"
-	diskformat         = "diskformat"
-	datastore          = "datastore"
-	StoragePolicyName  = "storagepolicyname"
+	// diskformat parameter is deprecated as of Kubernetes v1.21.0
+	diskformat        = "diskformat"
+	datastore         = "datastore"
+	StoragePolicyName = "storagepolicyname"
 
-	HostFailuresToTolerateCapability    = "hostfailurestotolerate"
-	ForceProvisioningCapability         = "forceprovisioning"
-	CacheReservationCapability          = "cachereservation"
-	DiskStripesCapability               = "diskstripes"
-	ObjectSpaceReservationCapability    = "objectspacereservation"
+	// hostfailurestotolerate parameter is deprecated as of Kubernetes v1.19.0
```

Comment:
Should these be v1.21 or are just adding comments to something that has already been deprecated?

Reply:
We do not want to remove any feature, so the deprecation notice is sufficient.